### ISSUE TITLE
cert problem

### ISSUE THREAD

**Issue by DimmoK** [RocketChat/Rocket.Chat.ReactNative#2591]
Hello, sorry for my bad English.
I have a problem. When I try to connect to my server that I see error for my cert. I have Android 10 and your app Rocket.Chat.

**Comment by diegolmello** [comment]
It's working on my tests.
Can you give us more info?

https://support.securly.com/hc/en-us/articles/212869927-How-to-install-Securly-SSL-certificate-on-Android-device-
https://www.ssldragon.com/blog/how-to-install-an-ssl-certificate-on-android/

**Comment by DimmoK** [comment]
Hello.Thanks for your help and sorry for my bad English.I haven't a cert file.I try connect to my workspace, for example:rchat.mysite.com:8866orhttps://rchat.mysite.com:8866And I see the error.I must get cert for this site?Thanks! Отправлено с моего телефона Huawei-------- Исходное сообщение --------От: Diego Mello <notifications@github.com>Дата: чт, 5 нояб. 2020 г., 17:13Кому: "RocketChat/Rocket.Chat.ReactNative" <Rocket.Chat.ReactNative@noreply.github.com>Копия: DimmKo <registration@deltamoby.ru>, Author <author@noreply.github.com>Тема: Re: [RocketChat/Rocket.Chat.ReactNative] cert problem (#2591)
It's working on my tests.
Can you give us more info?
https://support.securly.com/hc/en-us/articles/212869927-How-to-install-Securly-SSL-certificate-on-Android-device-
https://www.ssldragon.com/blog/how-to-install-an-ssl-certificate-on-android/

**Comment by diegolmello** [comment]
Yes. If the server is configured to require a client certificate, you have to install a certificate on your device.
Is that the case?

**Comment by DimmoK** [comment]
No. From browser I connect without any problem and doesn't request client cert. But I try from rchat app I have the problem.Отправлено с моего телефона Huawei-------- Исходное сообщение --------От: Diego Mello <notifications@github.com>Дата: чт, 5 нояб. 2020 г., 18:19Кому: "RocketChat/Rocket.Chat.ReactNative" <Rocket.Chat.ReactNative@noreply.github.com>Копия: DimmKo <registration@deltamoby.ru>, Author <author@noreply.github.com>Тема: Re: [RocketChat/Rocket.Chat.ReactNative] cert problem (#2591)
Yes. If the server is configured to require a client certificate, you have to install a certificate on your device.
Is that the case?

**Comment by diegolmello** [comment]
It might be an issue on your server SSL configuration then.
iOS and Android require more strict SSL settings in order to work.

**Comment by DimmoK** [comment]
Hello, maybe it is my problem too
[Google Play - user's comment](https://play.google.com/store/apps/details?id=chat.rocket.android&reviewId=gp%3AAOqpTOGVkeGVGupI2VQMg-lRYR0Ky2L2shzr5ih-HEicOet7299lF4kQDaySa1EdK_GVfDUEK5fqxtw4b8jcEw)

**Comment by MikolajKakol** [comment]
I have the same problem. Our rocket chat is under VPN. Do I really need to add a certificate to my device? Shouldn't I be able to say go with that certificate, I'm fine with it, as I can in browser?
